frame/auction.ts

```diff
@@ -23,11 +23,20 @@ import { FetchJsonStatus, tryFetchJson } from "./fetch";
  * used has no use for scoring signals, but when more algorithms are supported,
  * this data will be passed to them.
  *
+ * The request's `keys` query parameter consists of all the ads' rendering URLs,
+ * escaped and then joined with unescaped commas, ordered first alphabetically
+ * by interest group name and then by the order given in the
+ * `joinAdInterestGroup` call, with only the first kept in case of duplicates.
+ * This leaks some information about interest groups and when they were joined
+ * to the trusted server, but that's okay, since it's trusted.
+ *
  * @param hostname The hostname of the page where the FLEDGE Shim API is
  * running.
  */
 export async function runAdAuction(
   trustedScoringSignalsUrl: string | null,
+  // This is temporary until trustedBiddingSignalsUrl is added.
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
   hostname: string
```

Comment:
For what it's worth, I think Chrome probably is going to provide `hostname`

Reply:
Filed #103.